test/cri-containerd/helper_exec_test.go

```diff
@@ -78,12 +78,12 @@ func execInHost(ctx context.Context, client shimdiag.ShimDiagService, args []str
 }
 
 // shimDiagExecOutput is a small wrapper on top of execInHost, that returns the exec output
-func shimDiagExecOutput(ctx context.Context, tb testing.TB, podID string, cmd []string) string {
-	tb.Helper()
+func shimDiagExecOutput(ctx context.Context, t *testing.T, podID string, cmd []string) (string, error) {
```

Comment:
why the change from `testing.TB` to `*testing.T`?

Reply:
weird. not sure how that happened.